dojo/tools/trufflehog/parser.py

```diff
@@ -193,7 +193,8 @@ def get_findings_v3(self, data, test):
                     url="N/A",
                     dynamic_finding=False,
                     static_finding=True,
-                    nb_occurences=1
+                    nb_occurences=1,
+                    payload=rawV2
```

Comment:
I am not seeing `rawV2` used in any of the unit test files. Is this a new field? Would there be any concern for folks using older versions of truffle hog without `rawV2` in their scan reports? In those cases, the only field that would be used for deduplication would be the file path

Reply:
@Maffooch I had a further look on that, another way to do this would be using `url`. 

This field can be completed using Trufflehog v2 and Trufflehog v3.
Furthermore it's unique as it contains `path`, `commit_hash` and `line_number`.

We can also add the `component_name` to make sure we add no regression from before.

Reply:
With this update we make sure that any further change from Trufflehog or on the default description won't impact issue duplications

Reply:
As `url` was left empty until now, projects might have issues when upgrading DefectDojo.
Is there a migration mechanism so that I can write a script to fill `url` field for existing findings?